core/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -12,8 +12,7 @@ Implementation-Vendor-Id: org.mule.patterns
 Implementation-Vendor: MuleSoft, Inc.
 Build-Date: 2012-Apr-13 16:46:57
 Build-Revision: 24242
-Description: Mule ESB and Integration Platform
-Dev-List-Email: mule-esb@mulesoft.com

```

Comment:
I just removed the code that used this entry. Other than that this is a custom field.
As for the pom entry, it is entirely optional, and used only for the site generation through maven.